.github/workflows/benchmark.yaml

```diff
@@ -0,0 +1,53 @@
+name: Benchmarking Tests at Scale
+on:
+  pull_request:
```

Comment:
change it to `push` once this PR is good to go

Reply:
Not sure if we need to run this in every push, or on schedule

Reply:
like suggested https://github.com/envoyproxy/gateway/pull/3599#discussion_r1643460287, we can run this with `/benchmark` command.

Reply:
I think it is good to run it on pull/push. In general, I like all major testing/linting/etc. CI suites to run on every push even if there is no PR. Makes it easy to get your branches in order without having a PR that goes through a bunch of edits to get things working. I dislike the idea of only ever running it when users comment `/benchmark`. The general idea should be for CI to alert us when incoming changes degrade (or improve) performance.